packages/web-client/app/controllers/card-pay/deposit-withdrawal.ts

```diff
@@ -27,6 +27,12 @@ export default class CardPayDepositAndWithdrawalController extends Controller {
     this.workflowPersistenceId = null;
   }
 
+  get safes() {
+    return this.layer2Network.safes.value
+      ?.filter((s) => s.type === 'merchant' || s.type === 'depot')
+      .sortBy('type');
```

Comment:
Maybe a good time to introduce convenience getters for safes by types in the safe resource? I could follow up with this.

Reply:
yes, it could follow a pattern like `get issuePrepaidCardSourceSafes()` 🤔